tasks/section1.yml

```diff
@@ -342,7 +342,7 @@
       - rule_1.1.3
 
 - name: "SCORED | 1.1.4 | AUDIT | Ensure nosuid option set on /tmp partition"
-  command: /bin/true
+  shell: mount | grep /tmp | grep nosuid
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*